community-plugins.json

```diff
@@ -983,7 +983,7 @@
         "id": "media-extended",
         "name": "Media Extended",
         "author": "AidenLx",
-        "description": "Add extended media features for video and audio, including timestamp links and playback speed.",
+        "description": "Integrate, manage, and play video or audio directly in Obsidian. (Closed source)",
```

Comment:
Don't include "Obsidian" in the description.

Reply:
plugin description is updated :)